src/components/app/index.tsx

```diff
@@ -1,4 +1,6 @@
 import { h, Component } from 'preact';
+import { partial } from 'filesize';
```

Comment:
I don't know why I can't just import the library. I suspect the types are broken. Will file an issue to investigate and fix.